Solutions/SeraphicSecurity/Data/Solution_Seraphic_Security.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Name": "SeraphicSecurity",
+  "Author": "Seraphic Security - support@seraphicsecurity.con",
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/LogoSeraphicSecurity.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The Seraphic Web Security data connector provides the capability to ingest [Seraphic Web Security](https://seraphicsecurity.com/) events and alerts into Microsoft Sentinel.",
+  "Data Connectors": [
+    "Solutions/SeraphicSecurity/Data Connectors/SeraphicSecurityConnector.json"
+  ],
+  "Workbooks": [],
```

Comment:
If this solution does not contain workbooks, please remove the empty `"Workbooks":[]` filed.